main.tf

```diff
@@ -0,0 +1,204 @@
+module "label" {
+  source     = "github.com/cloudposse/terraform-terraform-label.git?ref=master"
+  attributes = "${var.attributes}"
+  delimiter  = "${var.delimiter}"
+  name       = "${var.name}"
+  namespace  = "${var.namespace}"
+  stage      = "${var.stage}"
+  tags       = "${var.tags}"
+}
+
+# Cloudwatch Log Group
+resource "aws_cloudwatch_log_group" "app" {
```

Comment:
this is not used anywhere in the module.
if it's supposed to be used from outside, add the outputs from the resource